src/tests/testCacheManager.cc

```diff
@@ -54,7 +54,7 @@ CacheManagerInternals::testValidUrl(const AnyP::Uri &url)
         (void)ParseUrl(url);
     } catch (...) {
         std::cerr << "\nFAIL: " << url <<
-            Debug::Extra << "error: " << CurrentException << "\n";
```

Comment:
Doesn't seem related to the PR

Reply:
This "Source Format Enforcement" PR (as all "Source Format Enforcement" PRs?) is the result of executing our `./scripts/source-maintenance.sh` script. That script may format sources and update CONTRIBUTORS. Both adjustment happened in this PR. Both are in-scope. Does this summary address your concerns?